3.2/root/usr/share/container-scripts/mongodb/init-petset-replset.sh

```diff
@@ -118,31 +70,28 @@ function add_member() {
   local replset_addr
   replset_addr="${MONGODB_REPLICA_NAME}/${endpoints}"
 
-  if ! mongo admin -u admin -p "${MONGODB_ADMIN_PASSWORD}" --host "${replset_addr}" --eval "${script}" --quiet; then
+  if ! mongo admin -u admin -p "${MONGODB_ADMIN_PASSWORD}" --host "${replset_addr}" --eval "while (!rs.add('${host}').ok) { sleep(100); }" --quiet; then
```

Comment:
DNS names are resolved to IP addresses before connection, so there is no difference. And IP addresses are used from same "source" as DNS names.

I wanted to remove ` # TODO: replace this with a call to `replset_addr` from common.sh, once it returns host names.` Because I think IP addresses and DNS names serves same in this usecase.